tests/python/frontend/pytorch/test_forward.py

```diff
@@ -1889,21 +1889,73 @@ def _get_default_vm_targets():
     return [tgt for (tgt, _) in tvm.testing.enabled_targets()]
 
 
-def verify_script_model(pt_model, ishapes, targets):
+def verify_script_model(pt_model, ishapes, targets, idtype=None):
     script_module = torch.jit.script(pt_model)
     verify_model_vm(script_module, ishapes, targets=targets)
 
+    if idtype:
+        verify_model_vm(script_module, ishapes, idtype=idtype, targets=targets)
+    else:
+        verify_model_vm(script_module, ishapes, targets=targets)
+
```

Comment:
I don't think you need this if/else, just `verify_model_vm(script_module, ishapes, idtype=idtype, targets=targets)` even if `idtype` is None should work.

Reply:
Done. I've cleaned up `verify_script_model()` and changed the handling of default arguments in `verify_model_vm()` so that passing in `None` will result in a `torch.float` being used.